examples/recursive-proofs/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "license": "UNLICENSED",
+  "scripts": {
```

Comment:
Can you add a script which generates everything necessary and then runs `main.ts`? We'll then want to add this to CI.